src/components/Button/buttonRules.ts

```diff
@@ -91,12 +98,15 @@ export default {
 
       ...(secondary && {
         color: typeSecondaryColor,
-        backgroundColor: typeSecondaryBackgroundColor,
+        backgroundColor: active
+          ? secondaryActiveStyle.backgroundColor
+          : typeSecondaryBackgroundColor,
         borderColor: typeSecondaryBorderColor,
         ':hover': {
           borderColor: 'transparent',
           backgroundColor: typeSecondaryBackgroundColorHover,
         },
+        borderWidth: `${!active ? (circular ? 1 : 2) : 0}px`,
```

Comment:
should this have the px-to-rem helper?

Reply:
I am not sure what that does. What do you suggest?

Reply:
@gopalgoel19, essentially it is about using `pxToRem()` helper function. You should think about whether this function should be used in your case or not - the rule is the following: if you would like your size to adapt font size changes (for example, if browser uses large font size for rendering page content you would like your sizes being proportionally larger as well), you should `rem` units instead of pixels. Seems that in your case `rem` units are more appropriate, as button's border should be thicker in case if font size is increased.

Please, use the following code snippet as an example (imageVariables.ts):
```js
export default () => ({
  ....
  avatarRadius: pxToRem(9999),
  avatarSize: pxToRem(32),
})
```